starlette/testclient.py

```diff
@@ -134,6 +134,7 @@ def send(
             scope = {
                 "type": "websocket",
                 "path": unquote(path),
+                "raw_path": path,
```

Comment:
The ASGI spec mandates this as bytes, so `path.encode()` I guess?

Reply:
You are correct. Fixed it.